tap_outbrain/__init__.py

```diff
@@ -385,6 +324,9 @@ def do_sync(args):
         LOGGER.fatal("Failed to generate a new access token.")
         raise RuntimeError
 
+    # NEVER RAISE THIS ABOVE DEBUG!
+    LOGGER.debug('Using access token `{}`'.format(access_token))
```

Comment:
Why log the access token? Is this an oversight?

Reply:
No you want the access token so you can work on the tap. Otherwise you're locked to two cycles per hour. https://amplifyv01.docs.apiary.io/#reference/rate-limits

We do not save the access token otherwise.